docs/Installation Guide.md

```diff
@@ -236,6 +236,13 @@ In addition to this, you would also have to populate the database for django-cit
 
     python manage.py migrate
     python manage.py cities_light
+    
+if ImportError: No module named cities_light ,this means cities_light has not installed properly.
+For this perform the following steps:
+
+    pip uninstall django-cities-light
+    pip install django-cities-light==3.2.0
+    pip install -r requirements.txt
```

Comment:
what happened is in 2-3 systems I saw django cities-light was not installed properly. So, the way to install it properly was it firstly uninstall the cities light that one was having , then install it using that command. then, again do pip install -r requirements.txt